scripts/apps/authoring/authoring/directives/AuthoringDirective.js

```diff
@@ -565,6 +565,36 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
              * in $scope.
              */
             $scope.publish = function(continueOnPublish) {
+                if (helpers.itemHasUnresolvedSuggestions($scope.item)) {
+                    modal.alert({
```

Comment:
Duplicates `/scripts/apps/authoring/authoring/directives/SendItem.js:71`. Can it be moved to one place?

Reply:
Done